samples/tab-deeplink-all-features/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-deeplink-all-features-nodejs",
+    "source": "officeDev",
+    "title": "DeepLink All Features",
+    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink all features",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-deeplink/csharp",
+    "longDescription": [
+      "Microsoft Teams sample app for demonstrating deeplink all features using tab"
```

Comment:
"longDescription": [
      "Microsoft Teams sample app for demonstrating deep links features using tab."
    ],

Reply:
fixed